lib/utils/helpers.ts

```diff
@@ -445,6 +445,18 @@ export const expandForAppName = {
 	},
 } as const;
 
+export const devicesSelectFields = {
+	$select: [
+		'id',
+		'uuid',
+		'device_name',
+		'status',
+		'is_online',
+		'supervisor_version',
+		'os_version',
+	] as any,
```

Comment:
```suggestion
	] satisfies keyof<BalenaSdk.Device>,
```

Reply:
I don't think we can do this as CLI is locked to TS 4.6.4 right now